assets/src/dashboard/components/layout/scrollable.js

```diff
@@ -52,10 +55,28 @@ const Scrollable = ({ children }) => {
   const {
     state: { scrollFrameRef, squishContentHeight },
   } = useLayoutContext();
+  const [scrollbarWidth, setScrollbarWidth] = useState(0);
+
+  const getScrollbarWidth = useCallback(() => {
+    const { current } = scrollFrameRef;
+
+    setScrollbarWidth(current.offsetWidth - current.clientWidth);
+  }, [scrollFrameRef]);
+
+  useEffect(() => {
+    if (!scrollFrameRef.current) {
+      return () => {};
+    }
+    getScrollbarWidth();
+
+    return () => {};
+  }, [getScrollbarWidth, scrollFrameRef]);
```

Comment:
I think this the callback and effect may not be worth it here.  Is there any reason we can't cut this block and do:
```jsx
    <ScrollContent ref={scrollFrameRef}>
      <Inner scrollbarWidth={scrollFrameRef.current.offsetWidth - scrollFrameRef.current.clientWidth} paddingTop={squishContentHeight}>
        {children}
      </Inner>
    </ScrollContent>
```

Reply:
We could do `scrollFrameRef?.current ? scrollFrameRef.current.offsetWidth - scrollFrameRef.current.clientWidth : 0` instead but need to check for current somehow because on initial load it's null.

Reply:
Yeah that's a good call 👍